plugins/promela/hu.bme.mit.gamma.promela.verification/src/hu/bme/mit/gamma/promela/verification/PromelaVerifier.xtend

```diff
@@ -101,10 +115,7 @@ class PromelaVerifier extends AbstractVerifier {
 			
 			// spin -t -p PromelaFile.pml
 			val traceCommand = '''spin -t -p -g -l -w «modelFile.canonicalPath.escapePath»'''
```

Comment:
Maybe we should use a string array with parameters instead of  a single string to prevent crashes in case spaces are used in a path (see e.g., Uppaal or Theta verifier).